src/Microsoft.Framework.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs

```diff
@@ -38,8 +38,8 @@ public CommandLineApplication(bool throwOnUnexpectedArg = true)
         public List<string> RemainingArguments { get; private set; }
         public bool IsShowingInformation { get; protected set; }  // Is showing help or version?
         public Func<int> Invoke { get; set; }
-        public Func<string> VersionGetter { get; set; }
-
+        public Func<string> LongVersionGetter { get; set; }
+        public Func<string> ShortVersionGetter { get; set; }
```

Comment:
This shouldn't be in the CommandLineApplication... Same about the one above. The command line application is used for subcommands too and they have nothing to do with version